### DISCUSSION TITLE
Reputation system

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#46] [Ideas]
With the #45, we want to allow users to create/publish/update bundles as time evolves. This will ultimately lead to a huge chaotic mess

We will probably need to start thinking about implementing a reputation system to limit certain types of contributions to larger / popular bundles or public watchlists

**Comment by mkitzmann** [comment]
This leads more into the direction of a social network. While I like the idea, I think this presents a whole new branch of problems which I would rather solve furher down the line (v2).

That being said, the idea of a reputation system might most easily be solved by a `follow` approach (like instagram). So I would follow certain peoples portfolio,watchlist or bundle and a person with a lot of followers would rank higher.

The concept of public or shared bundles with multiple contributors seems like a much larger problem.

**Reply by sadortun** [reply]
> easily be solved by a follow approach

Yes ! Simple and very effective !

> I would rather solve furher down the line (v2).

Ehehehe ! You seems to diskily to put everything in the MVP prototype 😀😀. But yes, it will come later on, but it may have some design consideration we need to take in account now!

**Reply by mkitzmann** [reply]
> > I would rather solve furher down the line (v2).
> 
> Ehehehe ! You seems to diskily to put everything in the MVP prototype . But yes, it will come later on, but it may have some design consideration we need to take in account now!

Definitly, more than happy to talk about future visions and the impliciations they have on the initial design!
